src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -7025,12 +7025,17 @@ def _apply_with_udtf_and_dynamic_pivot_along_axis_1(
                 pandas_labels=["partition_id"]
             )[0]
```

Comment:
needed to fix this here because of an edge case test where multiple sessions where present.

Reply:
just curious, why would there be a case where there's multiple sessions present?

Reply:
Not sure, I am following here the pandas tests, specifically `TestApplyLikeMethodsWithMultipleSessionsSnow1216902` where multiple sessions are tested.